test/test_readcalls.c

```diff
@@ -76,7 +84,7 @@ int pacc_pa(void) {
     return 0;
 }
 
-#define QSO1 " 40SSB 12-Jan-18 16:34 0006  PY9BBB         599  599  15                    10         \n"
+#define QSO1 " 80SSB 12-Jan-18 16:34 0006  PY9BBB         59   59   15            PY   15  3  14025.0\n"
```

Comment:
Use a valid log line. Moved to 80 as 40 is the default band and this may hide errors.